xlive/H2ConsoleCommands.cpp

```diff
@@ -465,6 +465,7 @@ void ConsoleCommands::handle_command(std::string command) {
 		}
 		else {
 			output(L"Unknown command.");
+			h2mod->logToDedicatedServerConsole(L"No such custom command.");
```

Comment:
i looked at code, output will output to console, u can just remove this

Reply:
alright, thanks